### DISCUSSION TITLE
Harvi value set to 0 W when offline

### DISCUSSION THREAD

**Discussion by mwitlox** [CJNE/ha-myenergi#447] [Ideas]
When the sun is gone, Harvi remains at its last value, so not 0. Possible to build in a check of Harvi is offline and put the value to 0

![chrome_akQkoQODJQ](https://github.com/CJNE/ha-myenergi/assets/96582378/ca1726a9-f3d2-4d51-aa0a-3a2369c32e33)

**Comment by silverailscolo** [comment]
To catch this, I added a 2 minutes "harvi elapsed timer" Helper, since only as long as any current is detected, harvi reports every 1 minute. That report resets the timer (and logs the time in an input_number).
Now whenever the timer runs out, it means we no longer get a report from myenergi and current is assumed to be 0, which I reset using an Automation.
